hack/lib/init.sh

```diff
@@ -44,7 +44,7 @@ KUBE_RSYNC_COMPRESS="${KUBE_RSYNC_COMPRESS:-0}"
 export no_proxy=127.0.0.1,localhost
 
 # This is a symlink to binaries for "this platform", e.g. build tools.
-THIS_PLATFORM_BIN="${KUBE_ROOT}/_output/bin"
+export THIS_PLATFORM_BIN="${KUBE_ROOT}/_output/bin"
```

Comment:
I confirmed that this is read (by golang.sh), the same applies for KUBE_OUTPUT_HOSTBIN, and KUBE_NONSERVER_GROUP_VERSIONS is read in multiple places